content/en/docs/examples/microservices-istio/add-new-microservice-version/index.md

```diff
@@ -44,14 +44,14 @@ tests, end-to-end tests and tests in a staging environment.
     1.  Send a request to the pod and see that it returns the correct result:
 
         {{< text bash >}}
-        $ kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') -- curl $REVIEWS_V2_POD_IP:9080/reviews/7
+        $ kubectl exec $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') -- curl $REVIEWS_V2_POD_IP:9080/reviews/7
```

Comment:
When the snip.sh files are generated for the automated tests, The lint checker is also going to complain about unquoted env variables,  so you might as well fix those here too.
```suggestion
        $ kubectl exec $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') -- curl "$REVIEWS_V2_POD_IP:9080/reviews/7"
```